### DISCUSSION TITLE
Access to email from Thunderbird is very slow, taking sometimes over 20 minutes to receive messages

### DISCUSSION THREAD

**Discussion by heisenberg2980** [forwardemail/forwardemail.net#299] [General]
I am accessing my email address using Thunderbird as recommended by the developers of forwademail, but the email is really slow to download new emails and take any action like marking emails as read/unread or moving emails to folders. When sending emails, a lot of times thunderbird says that the email has been sent but it hasn´t been possible to save it in the sent folder, and I have to retry multiple times or save it locally.

I opened issue https://github.com/forwardemail/forwardemail.net/issues/298 for this problem, and I believe this should be happening to everyone and not only me, is there anyone here experiencing the same issue?

**Comment by graemev** [comment]
I'm seeing very much delayed imap.  Right now it's 14:39 BST , my most recent email via imap (multiple clients , desktop & phone) is 12:01 and yet I can see I've downloaded mail (from forwardemail) via pop at 14:09 BST.

I've raised many tickets in this and related areas, I'm not seeing any kind of response ?  Feels like it's unmanned.

**Comment by titanism** [comment]
These issues should be resolved now.  If you continue to have errors or slowness let us know of the specific error code you receive (e.g. connection timeout) and as much detail/screenshots you can provide (email works best, send direct to `support@forwardemail.net`).
